marlin/src/constraints/verifier_key/prepared_circuit_verifier_key.rs

```diff
@@ -269,85 +269,6 @@ mod test {
 
     type MultiPCVar = SonicKZG10Gadget<Bls12_377, BW6_761, Bls12_377PairingGadget>;
 
-    #[test]
-    fn test_alloc() {
```

Comment:
This test was removed as `SonicKZG10::PreparedVerifierKeyVar::alloc_constant` should not be called any more. Instead, we should use `VerifierKeyVar::prepare` (and are using this)